tests_pytorch/test_attacks.py

```diff
@@ -0,0 +1,330 @@
+# pylint: disable=missing-docstring
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+from nose.plugins.skip import SkipTest
+import torch
+
+from cleverhans.devtools.checks import CleverHansTest
+from cleverhans.future.torch.attacks.fast_gradient_method import fast_gradient_method
+from cleverhans.future.torch.attacks.projected_gradient_descent import projected_gradient_descent
+
+class SimpleModel(torch.nn.Module):
+
+  def __init__(self):
+    super(SimpleModel, self).__init__()
+    self.w1 = torch.tensor([[1.5, .3], [-2, .3]])
+    self.w2 = torch.tensor([[-2.4, 1.2], [.5, -2.3]])
+
+  def forward(self, x):
+    x = torch.matmul(x, self.w1)
+    x = torch.sigmoid(x)
+    x = torch.matmul(x, self.w2)
+    return x
+
+class CommonAttackProperties(CleverHansTest):
+
+  def setUp(self):
+    # pylint: disable=unidiomatic-typecheck
+    if type(self) is CommonAttackProperties:
+      raise SkipTest()
```

Comment:
I put it there just in case we need to add some generic tests for all attacks under CommonAttackProperties. But I do agree that in the current implementation the ```if``` statement is not needed. Already removed it, thanks.